buildflow/runtime/ray_io/file_io.py

```diff
@@ -36,7 +36,7 @@ def actor(self, remote_fn: Callable, is_streaming: bool):
         return FileSinkActor.remote(remote_fn, self)
 
 
-@ray.remote
+@ray.remote(num_cpus=FileSink.num_cpus())
```

Comment:
optional idea: What if instead of saying num_cpus=IO.num_cpus(), we did: **IO.ray_args() so we only have to update the IO class?

Reply:
I think we can actually pass them when we instantiate the actor. Something like `PubsubSourceActor.options(...).remote()` then the actors won't have to worry about it. But definitely agreed annoying to update all the actor classes.